Mathlib/Topology/Order/Category/FrameAdjunction.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2023 Anne Baanen, Sam v. Gool, Leo Mayer, Brendan S. Murphy. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen, Sam v. Gool, Leo Mayer, Brendan S. Murphy
+-/
+import Mathlib.Topology.Category.Locale
+
+/-!
+# Adjunction between Locales and Topological Spaces
+
+This file defines functors between the categories of Locales and Topological Spaces
+and proves that they form an adjunction.
+
+## Main declarations
+
+* `pt`: the *points* functor from the category of locales to the category of topological spaces.
+* `topToLocale`: the forgetful functor from the category of topological spaces to the category of
+  locales.
+
+- `locale_top_adjunction`: the theorem that topToLocale is left adjoint to pt.
+
+## Motivation
+
+This adjunction provides a framework in which several Stone-type dualities fit.
+
+## Implementation notes
+
+- In naming the various functions below, we follow common terminology and reserve the word *point*
+  for an inhabitant of a type `X` which is a topological space, while we use the word *element* for
+  an inhabitant of a type `L` which is a locale.
```

Comment:
In the first line you used capital T, S, L for Topological Spaces and Locales, but quite consistently you write them with small letters from that point on. Perhaps all my comments should be forgotten and the first occurence is the one to change?